README.md

```diff
@@ -1,50 +1,102 @@
 # merchandise-in-baggage
 
-**Who uses the repo/service**
+## Who uses the repo/service
 
-This is the backend service supporting two frontends (public facing and internally for administration through stride).
+merchandise-in-baggage backend service supports two frontends (public facing and internally for administration through
+stride).
 
-Both frontend services are maintained from the [merchandise-in-baggage-frontend](https://github.com/hmrc/merchandise-in-baggage-frontend) repository
-and are deployed as separate instances `merchandise-in-baggage-frontend` & `merchandise-in-baggage-admin-frontend`.
+Both `merchandise-in-baggage-frontend` & `merchandise-in-baggage-admin-frontend`
+services are maintained from
+the [merchandise-in-baggage-frontend](https://github.com/hmrc/merchandise-in-baggage-frontend) repository
+and are deployed as separate instances.
 
-**How to start the service locally**
+## Start service locally
 
-`sbt run` This will only start the service as standalone but unable to interact with any other services including DataBase
+`sbt run` This will only start the service as standalone but unable to interact with any other services
+
+To complete a journey locally run the services via [service manager 2](https://github.com/hmrc/sm2)
+with the following profile:
 
-To complete a journey locally run the services (minus the frontend) via [service manager 2](https://github.com/hmrc/sm2) with the following profile:
 ```bash
 sm2 --start MERCHANDISE_IN_BAGGAGE_ALL
-sm2 --stop MERCHANDISE_IN_BAGGAGE
 ```
 
-`local base url and port` http://localhost:8280/
+### Service Local URL
+
+For admin facing you will need to login through stride with the roles found in `application.conf`
 
-**How to run tests**
+```
+http://localhost:8281/declare-commercial-goods/start-import 
+``` 
+
```

Comment:
Not sure why this is here also. Should be in the frontend as there is no login to stride in the backend and that is the frontend port. This might lead to confusion that the stride roles can be found in the backend's `application.conf`. Please remove, thanks.